example/data-persistence-tf/main.tf

```diff
@@ -36,7 +36,7 @@ module "provision_database" {
   permissions_boundary_arn               = var.permissions_boundary_arn
   rds_user_password                      = var.rds_user_password == "" ? random_string.db_pass.result : var.rds_user_password
   rds_connection_timing_configuration    = var.rds_connection_timing_configuration
-  dbRecreation                           = true
+  dbRecreation                           = false
```

Comment:
We should not make this change - DB recreation for int test projects is important.